pkg/tests/tasks/traffic/egress/egress_gateways_tls_file_mount_test.go

```diff
@@ -15,173 +15,198 @@
-	util.KubeDeleteContents("bookinfo", util.RunTemplate(nginxGatewayTLSTemplate, smcp))
-
-	util.Shell(`kubectl -n %s rollout undo deploy istio-egressgateway`, meshNamespace)
-	time.Sleep(time.Duration(20) * time.Second)
-	util.Shell(`oc wait --for condition=Ready -n %s smcp/%s --timeout 180s`, meshNamespace, smcpName)
-	util.Shell(`kubectl -n %s rollout history deploy istio-egressgateway`, meshNamespace)
-
-	util.Shell(`kubectl delete -n %s secret nginx-client-certs`, meshNamespace)
-	util.Shell(`kubectl delete -n %s secret nginx-ca-certs`, meshNamespace)
-	util.KubeDeleteContents("bookinfo", util.RunTemplate(ExGatewayTLSFileTemplate, smcp))
-	util.KubeDeleteContents("bookinfo", ExServiceEntry)
-	nginx.Uninstall()
-	sleep.Uninstall()
-	time.Sleep(time.Duration(20) * time.Second)
-}
-
 func TestTLSOriginationFileMount(t *testing.T) {
-	test.NewTest(t).Id("T14").Groups(test.Full, test.InterOp).NotRefactoredYet()
-
-	defer cleanupTLSOriginationFileMount()
-	defer util.RecoverPanic(t)
-
-	log.Log.Info("TestEgressGatewaysTLSOrigination File Mount")
-	sleep := examples.Sleep{Namespace: "bookinfo"}
-	sleep.Install()
-	sleepPod, err := util.GetPodName("bookinfo", "app=sleep")
-	util.Inspect(err, "Failed to get sleep pod name", "", t)
-
-	t.Run("TrafficManagement_egress_gateway_perform_TLS_origination", func(t *testing.T) {
-		defer util.RecoverPanic(t)
-
-		log.Log.Info("Perform TLS origination with an egress gateway")
-		util.KubeApplyContents("bookinfo", ExServiceEntry)
-		time.Sleep(time.Duration(10) * time.Second)
-
-		command := `curl -sSL -o /dev/null -D - http://istio.io`
-		msg, err := util.PodExec("bookinfo", sleepPod, "sleep", command, false)
-		util.Inspect(err, "Failed to get response", "", t)
-		if strings.Contains(msg, "301 Moved Permanently") {
-			log.Log.Info("Success. Get http://istio.io response")
-		} else {
-			log.Log.Infof("Error response: %s", msg)
-			t.Errorf("Error response: %s", msg)
-		}
-
-		log.Log.Info("Create a Gateway to external istio.io")
-		util.KubeApplyContents("bookinfo", util.RunTemplate(ExGatewayTLSFileTemplate, smcp))
-		time.Sleep(time.Duration(20) * time.Second)
-
-		command = `curl -sSL -o /dev/null -D - http://istio.io`
-		msg, err = util.PodExec("bookinfo", sleepPod, "sleep", command, false)
-		util.Inspect(err, "Failed to get response", "", t)
-		if strings.Contains(msg, "301 Moved Permanently") || !strings.Contains(msg, "200") {
-			log.Log.Infof("Error response: %s", msg)
-			t.Errorf("Error response: %s", msg)
-		} else {
-			log.Log.Infof("Success. Get http://istio.io response")
-		}
-
-		log.Log.Info("Cleanup the TLS origination example")
-		util.KubeDeleteContents("bookinfo", util.RunTemplate(ExGatewayTLSFileTemplate, smcp))
-		util.KubeDeleteContents("bookinfo", ExServiceEntry)
-		time.Sleep(time.Duration(20) * time.Second)
-	})
+	NewTest(t).Id("T14").Groups(Full, InterOp).Run(func(t TestHelper) {
+		hack.DisableLogrusForThisTest(t)
+		t.Log("Test TLS Origination with File Mount")
```

Comment:
This seems unnecessary, since it almost perfectly matches the name of the test. Can you replace this with a description of what the test does exactly (e.g. `t.Log("This test creates bla bla and checks bla bla")`?